src/main/java/com/unascribed/fabrication/features/FeatureLegacyCommandSyntax.java

```diff
@@ -58,7 +61,25 @@ public void apply() {
 							.executes(c -> DifficultyCommand.execute(c.getSource(), difficulty)));
 				}
 				dispatcher.register(diffCmd);
-				
+
+				dispatcher.register(CommandManager.literal("experience")
+						.requires(scs -> MixinConfigPlugin.isEnabled("*.legacy_command_syntax") && scs.hasPermissionLevel(2))
+						.then(CommandManager.argument("amount", IntegerArgumentType.integer())
+								.executes(c -> {
+									ServerPlayerEntity serverPlayerEntity = c.getSource().getPlayer();
+									int amount = IntegerArgumentType.getInteger(c, "amount");
+									serverPlayerEntity.addExperience(amount);
+									c.getSource().sendFeedback(new TranslatableText("commands.experience.add.points.success.single", amount, serverPlayerEntity.getDisplayName()), true);
+									return 1;
+								})).then(CommandManager.argument("lvlAmount", StringArgumentType.word()).executes((c) -> {
+							ServerPlayerEntity serverPlayerEntity = c.getSource().getPlayer();
+							String str = StringArgumentType.getString(c, "lvlAmount");
+							int amount = Integer.parseInt(str.substring(0,str.length()-1));
```

Comment:
You should make sure the string ends in L/l and throw a CommandSyntaxException if not instead of just blindly trimming the last character.